assets/js/directives/student_sidebar_directive.js

```diff
@@ -0,0 +1,11 @@
+app.directive("studentSidebarDirective", ["$route", "Module", function($route, Module) {
+  return {
+    templateUrl: "/js/templates/student_sidebar_template.html",
+    link: function(scope, elem, attr) {
+      scope.params = $route.current.params;
```

Comment:
you don't need to put variables on the scope object if you're not displaying them. also, you're not using this variable